COMMITTERS

```diff
@@ -33,3 +33,5 @@ Committer list:
 * Pirmin Vogel (vogelpi)
 * James Wainwright (jwnrt)
 * Alex Williams (a-will)
+* Pascal Nasahl (nasahlpa)
```

Comment:
This list should be kept alphabetical (by surname)

Reply:
Thanks, fixed